add tokenomics to application section (#1729)
hyperledger-labs/splice: docs/src/app_dev/overview/index.rst

Splice Daml APIs Overview
-------------------------

Splice implements several decentralized applications whose on-ledger state and workflows are implemented in Daml.  An `Interface <https://docs.digitalasset.com/build/3.3/reference/daml/interfaces.html#reference-interfaces>`__

> defined in Daml code is a public APIs that you should develop against because they are stable.

Reviewer: ```suggestion
defined in Daml code is a public API that you should develop against because they are stable.
```

Author: Done

These public APIs are used in interoperability standards,
like the :ref:`Canton Network Token Standard <app_dev_token_standard_overview>`.

> Use these Daml APIs to minimize the coupling between your application and your dependencies.  For example, the ``Interfaces`` in the Splice implementaiton loosely

Reviewer: ```suggestion
Use these Daml APIs to minimize the coupling between your application's Daml code and and the Daml code of your dependencies.  For example, the ``Interfaces`` in the Splice implementation loosely
```

couple an application with the implementation so that a Splice upgrade avoids forcing a corresponding upgrade of an application's Daml code.

See the :ref:`app_dev_daml_api` for an overview of the Daml APIs defined in Splice and their purpose.

Splice Daml Models Overview
---------------------------

A Daml model's `Templates <https://docs.digitalasset.com/build/3.3/reference/daml/templates.html>`__ and
`Choices <https://docs.digitalasset.com/build/3.3/reference/daml/choices.html>`__ are considered internal implementation details.  For example,

> the :ref:`Canton Network Token Standard <app_dev_token_standard_overview>` is the public API for working with tokens.

Reviewer: ```suggestion
the :ref:`Canton Network Token Standard <app_dev_token_standard_overview>` is the public API for working with tokens, including Canton Coin.
```

Author: Done
